api/src/main/java/io/strimzi/api/kafka/model/status/ListenerStatus.java

```diff
@@ -55,6 +56,16 @@ public void setAddresses(List<ListenerAddress> addresses) {
         this.addresses = addresses;
     }
 
+    @Description("The A list of public TLS keys which can be used to verify the identity of the server when connecting " +
```

Comment:
"The A list..." seems to be a typo.
Anyway isn't it better saying "TLS certificates" instead of "public TLS keys"?

Reply:
The problem is that our documentation is using certificate as a synonym for private key. We have things like _certificates and public keys_ on many places. That is why I used the public keys.

Reply:
Really? It's quite the opposite!

Reply:
For example here we start the chapter with it: https://strimzi.io/docs/master/full.html#kafka-listener-certificates-str

Reply:
Are you referring at this sentence? "You can provide your own server certificates and public keys" ... in this case it's wrong imho. We should try to fix this because a server certificates IS a public key (signed).

Reply:
I'm not sure I really agree. In my life the most common and understandable seemed to be certificate = public + private key. I usually prefer to be specific to make it easier and ytry to avoid certificate where possible. If you want I can change it here.

Reply:
I lived a different life maybe :-) ...  but certificate was always = signed public key. Why a certificate should be public + private key? It means that if you are delivering the certificate, you are delivering both public and private keys somehow? which is not the case.

Reply:
When you generate a public + private key couple and you need a certificate out of it, you generate a CSR providing only the public key. The CSR will be processed by signing your public key generating the certificate. Your private key is not involved at all.
Actually you could do encryption stuff without any sign of your public key (but of course in this case you cannot prove that a public key is yours, so the sign by a CA for example).

Reply:
A certificate _contains_ a public key (it also includes information about the associated entity and a digital signature from the issuer). I think we should fix the documentation which @scholzj linked (and any other places where the language is unclear). For this description I would at "A list of TLS certificates..." which is imho clear and correct and lets the reader keep whatever mental model of a TLS certificate they already have (correct or not).

Reply:
Let me know if we need to change the terminology used in "Kafka listener certificates".

Reply:
Comments on the description above:

which can be used > that can be used

...when connecting to the Kafka broker. Does that make sense?

Reply:
@laidan6000 I think we should reword: "You can provide your own TLS certificates for the following types of listeners:", but let's see how this discussion concludes before we go making any changes.

Reply:
Sorry, I've just seen the full description: ... when connecting to the given listener.

Sounds good to me.